zitilib/models/examples/infrastructure.go

```diff
@@ -25,6 +27,7 @@ func (self *infrastructureFactory) Build(m *model.Model) error {
 
 func (_ *infrastructureFactory) buildInfrastructure(m *model.Model) error {
 	m.Infrastructure = model.InfrastructureBinders{
+		func(m *model.Model) model.InfrastructureStage { return aws_ssh_keys0.Express() },
```

Comment:
Does this effectively stop me from being able to use the SSH key I already have with the `example` models?

Reply:
No, the component checks if you have a flag set in your config. Only if the flag is set will it manage keys.
I might also update it so that it can use an existing key and check for key existence before importing.

Reply:
Right, so I need to change my `bindings.yml` for it not to try and manage keys. Why wouldn't we just look for the presence of `ssh_key_name` instead? If the user has that set, then it doesn't try and manage the key?

Reply:
I have this in my `bindings.yml`:

```
credentials:
  aws:
    ssh_key_name: "mquigley"
```
If `ssh_key_name` is there, then don't try to create a new key?